attest/core/src/ias/verifier.rs

```diff
@@ -531,14 +532,16 @@ impl IasReportVerifier {
 
         let report_data = VerificationReportData::try_from(report)?;
 
-        if self
-            .or_verifiers
-            .iter()
-            .any(|verifier| verifier.verify(&report_data))
-            && self
+        if (self.and_verifiers.is_empty()
```

Comment:
`self.and_verifiers.is_empty()` should be necessary since `Iterator.all` returns `true` if the iterator is empty.

Reference: https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.all